src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAWindowsDesktopProject.cs

```diff
@@ -145,6 +148,29 @@ public void It_imports_when_targeting_dotnet_3()
             getValuesCommand.GetValues().ShouldBeEquivalentTo(new[] { "true" });
         }
 
+        [WindowsOnlyFact]
+        public void It_does_not_warn_when_targeting_net_framework()
+        {
+            var testDirectory = _testAssetsManager.CreateTestDirectory().Path;
+            var newCommand = new DotnetCommand(Log, "new", "wpf", "--no-restore");
+            newCommand.WorkingDirectory = testDirectory;
+            newCommand.Execute()
+                .Should()
+                .Pass();
+
+            // Set TargetFramework to net472
+            var projFile = Path.Combine(testDirectory, Path.GetFileName(testDirectory) + ".csproj");
+            var project = XDocument.Load(projFile);
+            var ns = project.Root.Name.Namespace;
+            project.Root.Elements(ns + "PropertyGroup").Elements(ns + "TargetFramework").Single().Value = "net472";
+            project.Save(projFile);
+
+            var buildCommand = new BuildCommand(Log, testDirectory);
+            buildCommand.Execute()
+                .Should()
+                .Pass();
```

Comment:
The test name says "It_does_not_warn".  Either add a check that there are no warnings, or change the test name to something like "It_builds_successfully_when_targeting_net_framework".